.github/actions/publish-manifest/action.yml

```
@@ -45,14 +45,25 @@ runs:
       shell: bash
       run: |
         set -euo pipefail
+
+        # Get the image index digest
+        # Eg: index_digest docker.stackable.tech/stackable/hello-world:0.0.1-SNAPSHOT-stackable0.0.0-dev
+        #
+        # Note: `docker manifest push` currently outputs the same hash, but `manifest` is experimental and the
+        # STDOUT is more likely to change than the structured output.
+        function index_digest {
+          docker buildx imagetools inspect --format '{{json .Manifest}}' "$1" | jq -r '.digest'
```

```suggestion
          docker buildx imagetools inspect --format '{{println .Manifest.Digest}}' "$1"
```

This can be simplified

---

Thanks for that. I spent longer than I'd like to admit trying to get that directly.

It is strange how `{{.Manifest}}` works without `println` (the default), yet `{{.Manifest.Digest}}` doesn't. In fact, it doesn't even error and uses the default. 🤯